modules/ROOT/pages/authentication-authorization/manage-users.adoc

```diff
@@ -1015,14 +1015,14 @@ With `ENCRYPTED`, the password string is expected to be in the format of `<encry
 ** `0` is the first version and refers to the `SHA-256` cryptographic hash function with iterations `1`.
 ** `1` is the second version and refers to the `SHA-256` cryptographic hash function with iterations `1024`.
 <5> Specifies whether the user must change their password at the next login.
-If omitted, the default is `CHANGE REQUIRED`.
+If the optional `SET PASSWORD CHANGE [NOT] REQUIRED` is omitted, the default is `CHANGE REQUIRED`.
 The `SET PASSWORD` prefix of the `CHANGE [NOT] REQUIRED` clause is only optional if it directly follows the `SET PASSWORD 'password'` clause and is not part of a `SET AUTH` clause.
 <6> The default for `SET STATUS` is `ACTIVE`.
```

Comment:
Both of these seem wrong for ALTER as they imply things change even if not given. That is not how alter works.

It might help to add a sentence up around line 984 about only the specified parts of the ALTER command gets altered, if any clause is omitted the user keeps the current value of that part.

Then I'd probably just remove `<6>` and update `<5>` to say
```
<5> Specifies whether the user must change their password at the next login.
If the optional `SET PASSWORD CHANGE [NOT] REQUIRED` is omitted when adding native auth to a user (either by first removing pre-existing native auth or if the user does not have native auth to start with), the default is `CHANGE REQUIRED`.
The `SET PASSWORD` prefix of the `CHANGE [NOT] REQUIRED` clause is only optional if it directly follows the `SET PASSWORD 'password'` clause and is not part of a `SET AUTH` clause.
```
just to actually capture the fact that we only have a default in one specific case